de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/ui/wizards/AddProjectToSessionWizard.java

```diff
@@ -86,7 +87,7 @@
     private final String remoteProjectID;
     private final String remoteProjectName;
 
-    private final AbstractIncomingProjectNegotiation negotiation;
+    private final AbstractIncomingReferencePointNegotiation negotiation;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make "negotiation" transient or serializable. ([squid:S1948](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1948))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWaWrfeQsSPV6ZXI_B9S&open=AWaWrfeQsSPV6ZXI_B9S)

[ISSUE_KEY]: AWaWrfeQsSPV6ZXI_B9S